index.html

```diff
@@ -57,178 +67,188 @@ <h3>LET’S CONNECT</h3>
-          <li class="margin-spacer">CSS</li>
-          <li class="margin-spacer">JavaScript</li>
-        </ul>
-        <button class="margin-spacer">See Project</button>
       </div>
       <div class="grid-item">
         <div class="image-wrapper">
           <img src="/images/tonica1.png" alt="work-image" />
         </div>
 
-        <h2>Tonic</h2>
-        <div class="work-details">
-          <p class="first-detail">CANOPY</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">Back End Dev</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">2015</p>
+        <div class="card-divider">
+          <h2>Tonic</h2>
+          <div class="work-details">
+            <p class="first-detail">CANOPY</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">Back End Dev</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li class="margin-spacer">CSS</li>
+            <li class="margin-spacer">JavaScript</li>
+          </ul>
+          <button class="margin-spacer">See Project</button>
         </div>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <ul>
-          <li>HTML</li>
-          <li class="margin-spacer">CSS</li>
-          <li class="margin-spacer">JavaScript</li>
-        </ul>
-        <button class="margin-spacer">See Project</button>
       </div>
       <div class="grid-item">
         <div class="image-wrapper">
           <img src="/images/tonic.png" alt="work-image" />
         </div>
-
-        <h2>Tonic</h2>
-        <div class="work-details">
-          <p class="first-detail">CANOPY</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">Back End Dev</p>
-          <div class="spacer margin-spacer"></div>
-          <p class="margin-spacer">2015</p>
+        <div class="card-divider">
+          <h2>Tonic</h2>
+          <div class="work-details">
+            <p class="first-detail">CANOPY</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">Back End Dev</p>
+            <div class="spacer margin-spacer"></div>
+            <p class="margin-spacer">2015</p>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>HTML</li>
+            <li class="margin-spacer">CSS</li>
+            <li class="margin-spacer">JavaScript</li>
+          </ul>
+          <button class="margin-spacer" type="button">See Project</button>
         </div>
```

Comment:
- [ ] Your project cards are too small for the layout. Kindly increase their size to match the Figma. This change was requested by the previous reviewer but you have reduced them way too much. Kindly take guidance from the figma
 **figam**
 
![width1](https://user-images.githubusercontent.com/99656687/236431364-2282da2d-ab65-478e-9065-0b292272ac5d.png)

**Your design**
![width](https://user-images.githubusercontent.com/99656687/236431434-0b22a53e-a064-488c-b36c-b4b057204e7e.png)